app/api/event_copyright.py

```diff
@@ -40,22 +40,33 @@ class EventCopyrightList(ResourceList):
 
     def query(self, view_kwargs):
```

Comment:
I don't think there is any use of query function at all in ResourceList if only allowed method is POSt